.github/workflows/continuous-integration.yml

```diff
@@ -143,41 +143,17 @@ jobs:
           npm ci --ignore-scripts --no-audit --prefer-offline --progress=false
       - name: Types
         run: npm run types
-  test-release:
-    name: Test release
+  auto-merge:
+    name: Auto merge Dependabot updates
     runs-on: ubuntu-latest
+    needs:
+      - format
+      - lint
+      - spellcheck
+      - test
+      - types
     steps:
-      - name: Checkout
-        uses: actions/checkout@v1
-        with:
-          fetch-depth: 1
-      - name: Setup Node.js
-        uses: actions/setup-node@v1
-        with:
-          node-version: 12
-      - name: Cache dependencies
-        uses: actions/cache@v1
+      - name: auto-merge
```

Comment:
This should be `Auto merge`. Isn't it fixed in the Readme?

Reply:
copied it from other repo, updated